test-cases/longevity/longevity-5000-tables.yaml

```diff
@@ -40,3 +40,6 @@ store_results_in_elasticsearch: False
 # so we temporary removed the default `--abort-on-lsa-bad-alloc 1 --abort-on-seastar-bad-alloc`
 # once the issue would be solved we can enable it back.
 append_scylla_args: '--blocked-reactor-notify-ms 500'
+
+# TODO: remove when https://github.com/scylladb/scylla-tools-java/issues/175 resolved
```

Comment:
@roydahan I think it's relevant to the 1000 keyspaces

Reply:
in addition to 5000 tables?

Reply:
added